.github/workflows/discord-i18n-ping.yml

```diff
@@ -0,0 +1,35 @@
+name: Translation Tuesday
+
+on:
+  schedule:
+    - cron:  '0 0 * * TUE'
+  workflow_dispatch:
+
+jobs:
+  translation:
+    name: translation_tuesday
+    if: ${{ github.repository_owner == 'withastro' && github.event.commits[0].message != '[ci] format' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Install Tools & Dependencies
+        uses: ./.github/actions/install
+
+      - id: message
+        name: Format Discord message
+        env:
+          COMMIT_AUTHOR: ${{ github.event.commits[0].author.name }}
+          COMMIT_MESSAGE: ${{ github.event.commits[0].message }}
+          COMMIT_ID: ${{ github.event.commits[0].id }}
```

Comment:
```suggestion
```
These lines should not be required for the script as it doesn't access these ENV variables.